src/main/kotlin/app/revanced/patches/youtube/misc/video/quality/patch/RememberVideoQualityPatch.kt

```diff
@@ -43,18 +43,17 @@ class RememberVideoQualityPatch : BytecodePatch(
     override fun execute(context: BytecodeContext): PatchResult {
         SettingsPatch.PreferenceScreen.MISC.addPreferences(
             SwitchPreference(
-                "revanced_remember_video_quality_last_selected",
+                "revanced_video_remember_last_selected",
```

Comment:
This new key is not really suitable, considering that there is a remember playback and quality preference, they would conflict.

Reply:
"quality" was omitted, similarly "speed" would be. Therefor both keys equal.
Simply don't omit them to fix the issue: "revanced_remember_video_quality_last_selected".